### ISSUE TITLE
javascript injection

### ISSUE THREAD

**Issue by MacaylaMarvelous81** [RohanGhalib/publicnotepad#3]
It's possible to add notes with arbitrary HTML which can run potentially unsafe scripts. For example, [this note I made](https://rohanghalib.me/publicnotepad/view_note.php?id=82) runs `alert(location.href)`.

**Comment by RohanGhalib** [comment]
Right! i have added html sanitization so its no longer possible to add legacy html tags apart from those used in rich text formatting
